src/services

```diff
@@ -20,19 +20,16 @@ p({
   "name":"AuthenticatedNSpecDAO",
   "serve":true,
   "serviceScript":"""
-    return new foam.dao.PMDAO(x,
-      new foam.nanos.auth.AuthorizationDAO.Builder(x)
-        .setDelegate(x.get("nSpecDAO"))
-        .setAuthorizer(new foam.nanos.auth.StandardAuthorizer("service"))
-        .build()
-    );
+    return new foam.dao.EasyDAO.Builder(x)
+      .setPm(true)
+      .setAuthorizer(new foam.nanos.auth.StandardAuthorizer("service"))
+      .setInnerDAO(x.get("nSpecDAO"))
+      .setOf(foam.nanos.boot.NSpec.getOwnClassInfo())
+      .build();
   """,
   "client":"""
     {
-      "of":"foam.nanos.boot.NSpec",
-      "cache":true,
-      "ttlSelectPurgeTime": 0,
-      "ttlPurgeTime": 0
+      "of":"foam.nanos.boot.NSpec"
```

Comment:
Why removing the cache?

Reply:
Using the CacheDAO instead of the TTLCacheDAO causes an error that I do not understand. If you have time to fix it, please checkout the Medusa branches and then try to load the daoKey in a new DIG entity. Otherwise, this fixes it (we could check this in as an interm fix).

Reply:
@nanoscott So issue is classes referenced in nspec configuration not found js bin.  foam-bin.js 
For example foam.box.socket.SocketServer.  This is a model intended for server side only.

Reply:
@nanoscott if you add the caching back in, we'll merge the other changes.

Reply:
re-added.